web_timeline/static/src/js/timeline_controller.js

```diff
@@ -39,7 +39,7 @@ odoo.define('web_timeline.TimelineController', function (require) {
         update: function (params, options) {
             this._super.apply(this, arguments);
             if (_.isEmpty(params)){
-                return;
+                return $.Deferred().resolve();
```

Comment:
You can do `return $.when()`, which is the same. Possibly not needed anyways due to what is explained above.

Reply:
I now just return the deferred from the super.

Reply:
OK, just to be sure: The operation you're performing here depends somehow in the result from super?

Reply:
No. The update function is now done when the super and the reloading of records is done.